coding/sha1.hpp

```diff
@@ -14,10 +14,5 @@ class SHA1
 
   static Hash Calculate(std::string const & filePath);
   static std::string CalculateBase64(std::string const & filePath);
-
-  static Hash CalculateForString(std::string const & str);
```

Comment:
Please, keep one of the _buffer_ functions here and add a smoke test with some input buffer and known output.
Strange that we didn't have one ..